README.md

```diff
@@ -1 +1,68 @@
 # cucumber-js-appium-app-browserstack
+
```

Comment:
Do we want to add browserstack logo like in serenity?

https://github.com/browserstack/serenity-appium-app-browserstack/tree/sample-scripts

Reply:
Kept it consistent across nodejs repo